src/client/ws/transport.rs

```diff
@@ -254,7 +241,7 @@ impl<'a> WsTransportClientBuilder<'a> {
 	}
```

Comment:
`with_url`, `with_origin_header` and `with_timeout` are not used it seems, can delete?

Reply:
I prefer the keep those if we decide to make the `client generic` or expose more `builder options` but I could export those to "silence" the warnings.

Reply:
I see. Or just add `#[allow(unused)]`.

Reply:
Fixed in https://github.com/paritytech/jsonrpsee/pull/171/commits/1f904e1a4ab2d9775832b00931e22215ee0327ff